### ISSUE TITLE
Assemblies are not Strongly Named, hence unusable to me in current form

### ISSUE THREAD

**Issue by peSHIr** [bamcis-io/GeoJSON#9]
I tried to reference your NuGet package, but unfortunately (as my C# solution consists of Strongly Named assemblies), all I get at runtime isthe following exception:

System.IO.FileLoadException: 'Could not load file or assembly '[name], Version=#.#.#.#, Culture=neutral, PublicKeyToken=null' or one of its dependencies. A strongly-named assembly is required. (Exception from HRESULT: 0x80131044)'

Could you please update your package to be strongly named? Or else I see no other option to include some of your source code in my own instead of referencing your NuGet package, which sort of defeats the whole purpose, doesn't it...?

**Comment by hakenmt** [comment]
Yes, I think this is something I can implement in this package. I'll need some time to test and see if any of the dependencies of the solution are not also strong named to ensure it works, but the implementation is fairly straight forward. For reference, what is your use case for using strong named assemblies in your solution?

**Comment by peSHIr** [comment]
Really no desperate need (so far), except that it's "the right thing to
do". It would enable side by side installation and precise version ing (if
ever needed) I believe. Plus, the solution was started with strongly named
assemblies and consists of dozens of projects now, so turning it off would
be a pain.

For now I went with including source from GitHub into my little project
that needs to GeoJSON, because I have no other simple option. And I went
for GeoJSON.NET because of the higher use statistics. If your package would
be strongly named, I would definitely consider throwing the code copy out
and using an actual NuGet reference though.

Thanks for the quick reply.

Jarno Peschier

Op do 25 apr. 2019 03:10 schreef Michael Haken <notifications@github.com>:
